src/app/genes/gene-search/gene-search.component.html

```diff
@@ -24,11 +24,13 @@
             </div>
         </div>
         <ul *ngIf="errorMessage"  class="filter-select">
-            <li class="filter-select-list" [class.italic-result]="isNotFound">
+            <li class="filter-select-error" [class.italic-result]="isNotFound">
                 <div class="gene-name row">
-                    <div class="col-md-12 col-sm-12 col-12">
-                        {{errorMessage}}
-                    </div>
+                    <span class="gene-search-error">
```

Comment:
Bit dangerous putting this between the bootstrap grid definitions (row and col-*).  Does it still work if you put span inside the col div (a child of the <div class="col-md-12 col-sm-12 col-12" /> instead)?

Reply:
I moved the span inside the div, and tweaked a few things to get it working.